sample-status-updater-infrastructure/pom.xml

```
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>sample-status-updater-cli</artifactId>
         <groupId>life.qbic</groupId>
-        <version>1.3.0-SNAPSHOT</version>
+        <version>1.2.4</version>
```

```suggestion
        <version>1.3.0-SNAPSHOT</version>
```
We need to stay with the current version